package.json

```diff
@@ -27,23 +27,22 @@
 	],
 	"scripts": {
 		"postinstall": "npm run bootstrap",
-		"prettier:check": "prettier --check \"packages/*/src/**/*.ts\"",
 		"prettier:write": "prettier --write \"packages/*/src/**/*.ts\"",
-		"postpublish": "lerna run install:safe --scope lit-plugin && echo \"🎉 Published successfully! You can now publish the vscode extension seperately by running 'cd packages/vscode-lit-plugin && npm run publish'\"",
+		"postpublish": "echo \"🎉 Published successfully! You can now publish the vscode extension seperately by running 'cd packages/vscode-lit-plugin && npm run publish'\"",
 		"publish": "lerna run prepare && lerna publish --exact && npm run postpublish",
 		"publish:next": "lerna run prepare && echo \"When asked, please choose 'Custom Prerelease' and use default value\" && lerna publish --dist-tag next --preid next --exact && npm run postpublish",
-		"bootstrap": "lerna clean --yes && lerna bootstrap && lerna run install:safe --scope lit-plugin",
```

Comment:
finally got rid of that pesky thing :tada: 

just to check i understand properly - we now install everything as normal, then bundle the plugin sources _with_ dependencies and thats why we can get rid of the copylink/install:safe stuff. but we still copy tsc etc since vsc wants it.

Reply:
Yes, with the caveat that it's not quite install as normal because we let lerna do its thing, so there's a few symlinks flying around between the packages

And to build the vscode plugin we create a directory at `./packages/vscode-lit-plugin/built` that contains only and exactly those files that are needed, which is very few because we're also using esbuild to bundle almost all dependencies. Our tests of the plugin, as well as the package script, only uses that `built` directory